include/dlaf/sender/transform_mpi.h

```diff
@@ -23,13 +23,18 @@
 
 namespace dlaf::comm::internal {
 
-/// This helper "consumes" a CommunicatorPipelineExclusiveWrapper ensuring that after this call
-/// the one passed as argument gets destroyed. All other types left as they are
-/// by the second overload.
+/// This helper "consumes" a CommunicatorPipelineSharedWrapper or a CommunicatorPipelineSharedWrapper
```

Comment:
```suggestion
/// This helper "consumes" a CommunicatorPipelineExclusiveWrapper or a CommunicatorPipelineSharedWrapper
```

Reply:
this is a new level of error 😂 new achievement ✅

thanks for spotting it